eq-author-api/migrations/index.js

```diff
@@ -1,3 +1,10 @@
 const addVersion = require("./addVersion");
 
-module.exports = [addVersion];
+const migrations = [addVersion];
+
+const version = migrations.length;
+
+module.exports = {
+  migrations,
+  version,
```

Comment:
Can we rename version to be clearer about what we mean e.g. `latestVersion` or `currentVersion` or similar?

Reply:
I named it `version` so it was easier to add to objects but happy to rename for clarity 👍